src/components/Routes.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+
+import SocialMediaAuth from './SocialMediaAuth';
 import Home from './Home';
+import Signup from './Signup';
+import Profile from './Profile';
 
-export const Routes = () => (
+const Routes = ({ isAuth }) => (
   <Switch>
-    <Route exact path="/" render={props => <Home {...props} />} />
+    <Route exact path="/" render={props => <Home {...props} isAuth={isAuth} />} />
```

Comment:
This route should not be protected, remove `isAuth` middleware!

Reply:
this can be useful to display some contents for an authenticated user even though it's a public route